utam-core/src/main/java/utam/core/element/Element.java

```diff
@@ -74,6 +75,23 @@ public interface Element {
    */
   void click();
 
+  /**
+   * perform a secondary-button click on the element
+   */
+  void rightClick();
+
+  /**
+   * click and hold the element
+   * @param duration the duration for which to hold.
+   *                 Using Duration.ZERO will hold the click indefinitely
+   */
+  void clickAndHold(Duration duration);
```

Comment:
to align with JS https://github.com/salesforce-experience-platform/utam-js/pull/409/files let's use integer as parameter (duration in seconds)